docs/GUIDE_TO_KOGITO_DEVS.md

```diff
@@ -0,0 +1,174 @@
+# Guide for Core/Runtimes team to Smoke test local changes on Openshift/k8s Cluster.
+
+This guide aims to help core/runtimes developer to test their local changes with respect to the operator.
+The operator will be deployed over an Openshift or Kubernetes cluster. In this example we've taken minikube to deploy a Kubernetes cluster.
+
+# Table of contents
+
+* [Install Minikube](#install-minikube)
+* [Build Artifacts](#build-artifacts)
+* [Build the Kogito Application Images](#build-the-kogito-application-images)
+* [Build the Kogito Service Image](#build-the-kogito-service-image)
+* [Install the Operator](#install-the-operator)
+* [Run Kogito Service with Kogito Operator](#run-kogito-service-with-kogito-operator)
+
+## Install Minikube
+
+We decided to go with Minikube as the Kubernetes cluster as it is very resource efficient and can be started easily on any system.
+ 
+ ### Prerequisites
+ 
+  * Install `kubectl` binaries on your system [see](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+  * Have a hypervisor installed (kvm  is recommended for linux)
+
+For installing the Minikube cluster please follow [this tutorial](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+
+## Build Artifacts
+
+Before proceeding, please read the README files on the following repositories to know the dependencies you might need to install to build and deploy the artifacts.
+ * [kogito-runtimes](https://github.com/kiegroup/kogito-runtimes)
+ * [kogito-apps](https://github.com/kiegroup/kogito-apps)
+ * [kogito-examples](https://github.com/kiegroup/kogito-examples)
+ 
+To generate the jar artifacts, you can just run the following command on each repository:
+
+```shell-script
+mvn clean package -DskipTests #(For skipping tests)
```

Comment:
Can you please add `-DskipTestsIT`? We bound them for now to `verify` phase so it is not really needed but it is safer :)

Reply:
Added